base/shell.jl

```diff
@@ -312,3 +312,69 @@ julia> println(shell_escaped_winsomely("A B\\", "C"))
 """
 shell_escape_winsomely(args::AbstractString...) =
     sprint(print_shell_escaped_winsomely, args..., sizehint=(sum(length, args)) + 3*length(args))
+
+function print_shell_escaped_CMDly(io::IO, arg::AbstractString)
+    any(c -> c in ('\r', '\n'), arg) && throw(ArgumentError("Encountered unsupported character by CMD."))
+    # include " so to avoid toggling behavior of ^
+    arg = replace(arg, r"[%!^\"<>&|]" => s"^\0")
```

Comment:
```suggestion
    arg = replace(arg, r"[%)!^\"<>&|]" => s"^\0")
```
Upon closer inspection of https://stackoverflow.com/questions/4094699/how-does-the-windows-command-interpreter-cmd-exe-parse-scripts/4095133#4095133, I realized that `)` is a metacharacter even though `(` is not. (or we can include both here for symmetry)